app/views/layouts/_alert.html.haml

```diff
@@ -1,5 +1,6 @@
 .alert-box{ class: "#{type}" }
   = message
   - if local_assigns[:unconfirmed]
-    %a{ "data-action": "login-modal#resend_confirmation", "data-tab": local_assigns[:tab] }
```

Comment:
I believe we can now remove `resend_confirmation`, could you double check ?

Reply:
It was removed in a later commit.